plugins/woocommerce-admin/client/marketplace/components/likert-scale/likert-scale.scss

```diff
@@ -0,0 +1,51 @@
+.woocommerce-marketplace__likert-scale {
+	display: flex;
+	list-style: none;
+	margin: 0;
+	border: 2px solid transparent;
+
+	&.validation-failed {
+		border-color: var(--wc-red);
+	}
+}
+
+.woocommerce-marketplace__likert-scale-item {
+	margin: 0;
+	width: 100%;
+	font-size: 11px;
+
+	label {
+		align-items: center;
+		aspect-ratio: 91.6 / 48;
+		border: 1.5px solid transparent;
+		box-sizing: border-box;
+		display: flex;
+		flex-direction: column;
+		gap: 8px;
+		justify-content: center;
+		padding: 16px 8px;
+		text-align: center;
+	}
+
+	input:checked + label {
+		border-color: var(--wp-admin-theme-color-darker-10);
+	}
+
+	// Improve a11y (especially keyboard navigation) by transferring the outline from the (now hidden) checkbox to its label
+	input:focus {
+		margin-top: -99px;
```

Comment:
🗒️  **Hacky!** Necessary to position the "focus" outline out-of-sight (we replace it with our own focus outline), _without_ setting the checkbox to `display: none` (we need it "visible" so keyboard users can tab-through to the controls).